### PR TITLE
SCMOD-12263: Upgrade autonomy-aci-client-api to latest

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-12263

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -115,9 +115,9 @@
                 <version>1.0.0</version>
             </dependency>
             <dependency>
-                <groupId>com.autonomy.aci.client</groupId>
-                <artifactId>autonomy-aci-client-api</artifactId>
-                <version>4.1.3</version>
```
Comment (dermot-hardy) [inline pom.xml:120]: Update to the next major version - as this is dependency management, dropping this is a breaking change.

Comment (xbreizh) [reply pom.xml:120]: downgrading to 5.0.0

Comment (dermot-hardy) [reply pom.xml:120]: How does that help?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
Downgrading to 5.0.0 doesn't make removing `com.autonomy.aci.client:autonomy-aci-client-api` not a breaking change.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
I thought you were adding `com.hp.autonomy.aci.client:aci-api:12.4.0`.  I don't understand what you're doing here.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
All I should be seeing here is the addition of a new dependency.  Remove the whitespace changes from the `autonomy-aci-client-api` dependency.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**